### ISSUE TITLE
experiment with a model for record type categories

### ISSUE THREAD

**Issue by josh-chamberlain** [Police-Data-Accessibility-Project/data-source-identification#42]
How much better does the model perform if, instead of asking it for `record_type`, we ask it for `record_type_category`? We can use the existing data for this. It also makes annotation much easier.

The more granular record types are _useful_ for people, but we may be able to get by without them.

https://docs.pdap.io/activities/data-dictionaries/record-types-taxonomy

**Comment by mbodeantor** [comment]
Probably easiest to merge the current labeled data to one that associates each record_type to a record_type_category and use the record_type_category as the label instead.

@josh-chamberlain Does this association exist somewhere already or would we need to create it?

**Comment by josh-chamberlain** [comment]
@mbodeantor it doesn't exist anywhere but that doc, nowhere in the data.

**Comment by bonjarlow** [comment]
I've made a map for labels -> label_category, the updated csv of [url, label, label_category,...] will be up on hugging face or anywhere else you'd like it

**Comment by josh-chamberlain** [comment]
@bonjarlow do you want to link your Hugging Face work to this / do you think there's anything else to do before calling this close? we can always continue to refine the model, but I think you did it

**Comment by bonjarlow** [comment]
Agreed, conclusion is that record type category (coarse label) performs better than fine labels

https://huggingface.co/PDAP/coarse-url-classifier
